charts/civil-damages-service/values.preview.template.yaml

```diff
@@ -45,12 +45,14 @@ java:
         - robotics-notification-sender
         - robotics-notification-recipient
         - launch-darkly-sdk-key
+        - postgres-user
+        - postgres-password
```

Comment:
we don't have to pull it here, it's not used in our code

Reply:
hangover from playing around, good spot